app/assets/stylesheets/osem.css.scss

```diff
@@ -89,4 +89,15 @@ p.comment-body {
 
 .changeset{
   display: none;
-}
\ No newline at end of file
+}
+
+.quantity_left{
+  height: 3em;
+  width: 9em;
+}
+
+@media (max-width: 767px){
```

Comment:
Previously required to make buttoms responsive but as we have added `.visible-sm.visible-xs` class It is no longer required! Removed it.